mobile-widgets/qmlmanager.cpp

```diff
@@ -850,6 +850,7 @@ static void setupDivesite(DiveSiteChange &res, struct dive *d, struct dive_site
 		res.location = location;
 	} else {
 		res.createdDs.reset(alloc_dive_site_with_name(locationtext));
+		res.createdDs.get()->location = location;
```

Comment:
You can remove the `get()` here. You can directly dereference a `unique_ptr` with `->` or `*`.

`get()` is only needed if you explicitly want to transform the smart pointer into a plain pointer.